index.html

```diff
@@ -443,9 +443,11 @@
         WHATWG URL standard [[!URL]].
       </p>
       <p>
-        The term  <dfn><a href=
+        The terms <dfn><a href=
         "https://html.spec.whatwg.org/multipage/webappapis.html#concept-current-everything">
```

Comment:
Should go to https://html.spec.whatwg.org/multipage/webappapis.html#current-settings-object